app/assets/stylesheets/shared/_boxed_group.scss

```diff
@@ -0,0 +1,34 @@
+/* Blatently taken from Github's settings page */
+.boxed-group {
+  position: relative;
+  border-radius: 3px;
+  margin-bottom: 30px;
+
+  h3 {
+    background-color: #f5f5f5;
```

Comment:
Color literals like `#f5f5f5` should only be used in variable declarations; they should be referred to via variable everywhere else.